src/controllers/blob-storage.ts

```diff
@@ -7,14 +7,10 @@ import {
     StorageSharedKeyCredential
 } from '@azure/storage-blob';
 import * as dotenv from 'dotenv';
-import pino from 'pino';
 
-dotenv.config();
+import { logger } from '../utils/logger';
 
-export const logger = pino({
-    name: 'StatsWales-Alpha-App: BlobStorageService',
-    level: 'debug'
-});
```

Comment:
I've just realised I might've removed some clarity here by reusing the configured logger - [Child loggers](https://github.com/pinojs/pino/blob/main/docs/child-loggers.md) might be a solution here.

Reply:
done 👍